rpc/client/localclient.go

```diff
@@ -108,3 +113,40 @@ func (c Local) Validators() (*ctypes.ResultValidators, error) {
 func (c Local) Tx(hash []byte, prove bool) (*ctypes.ResultTx, error) {
 	return core.Tx(hash, prove)
 }
+
+func (c Local) Subscribe(query string, out chan<- types.TMEventData) error {
```

Comment:
iterate on this interface. is this the best interface? maybe callback style is better here? https://godoc.org/cloud.google.com/go/pubsub#Subscription.Receive

Also, pass context!